ethereum/p2p/src/test/java/tech/pegasys/pantheon/ethereum/p2p/discovery/PeerDiscoveryAgentTest.java

```diff
@@ -73,7 +73,7 @@ public void neighborsPacketLimited() {
 
     // Start another peer pointing to those 20 agents.
     final MockPeerDiscoveryAgent agent = helper.startDiscoveryAgent(otherPeers);
-    assertThat(agent.getPeers()).hasSize(20);
+    assertThat(agent.getPeers()).contains(otherPeers.toArray(new DiscoveryPeer[20]));
```

Comment:
Might be good to add a comment here related to this change.

Reply:
Done